wp-includes/sqlite/class-wp-sqlite-pdo-user-defined-functions.php

```diff
@@ -224,126 +224,99 @@ public function dateformat( $date, $format ) {
 	/**
 	 * Method to extract the month value from the date.
 	 *
-	 * @param string representing the date formatted as 0000-00-00.
+	 * @see https://www.php.net/manual/en/datetime.format.php
```

Comment:
This shims a MySQL function using SQLite datatype, the argument type is not specific to PHP so I don't think linking to PHP manual here is helpful. I intentionally included database-related information in the method comment and PHP-related information in a second comment inside the function body – this way a developer may learn both about the function inputs and the translation process.

Reply:
Ditto for all other functions in this class. 

Would you rather debug this code?

```php
	/**
	 * Method to extract the year value from the date.
	 *
	 * @see https://www.php.net/manual/en/datetime.format.php
	 *
	 * @return string representing the number of the year.
	 */
	public function year( $field ) {
		return intval( gmdate( 'Y', strtotime( $field ) ) );
	}
```

Or this code?

```php
	/**
	 * Method to extract the year value from the date.
	 *
	 * @param string representing the date formatted as 0000-00-00.
	 *
	 * @return string representing the number of the year.
	 */
	public function year($field)
	{
		/*
		 * From https://www.php.net/manual/en/datetime.format.php:
		 * 
		 * Y - A full numeric representation of a year, 4 digits.
		 */
		return intval(date('Y', strtotime($field)));
	}
```

The latter offers all the assumption about the inputs, translation, and outputs. The former only gives me limited information about the expected output.

Reply:
Fair point 👍 
Restored the comments.